pkg/security/module/policy_monitor.go

```diff
@@ -130,6 +130,7 @@ type RuleSetLoadedReport struct {
 func ReportRuleSetLoaded(sender EventSender, statsdClient statsd.ClientInterface, ruleSet *rules.RuleSet, err *multierror.Error) {
 	rule, event := NewRuleSetLoadedEvent(ruleSet, err)
 
+	// TODO(celia): Create new metric for threat score rules
```

Comment:
As the threat score rules are part of the default policy what kind of metric do you want to introduce ?